ansible_builder/main.py

```diff
@@ -371,7 +401,11 @@ def prepare_build_context(self):
 
     def prepare_galaxy_install_steps(self):
         if self.definition.get_dep_abs_path('galaxy'):
-            self.steps.extend(GalaxyInstallSteps(CONTEXT_FILES['galaxy']))
+            self.steps.extend(GalaxyInstallSteps(CONTEXT_FILES['galaxy'],
+                                                 self.copied_galaxy_keyring,
+                                                 self.galaxy_ignore_signature_status_code,
+                                                 self.galaxy_required_valid_signature_count))
```

Comment:
Mentioned in a comment elsewhere, but feels like we should probably verify `galaxy-keyring` is given if one or both of these other options are supplied. I don't think they're valid with it.